### ISSUE TITLE
Describe how to use `--dart-define-from-file`

### ISSUE THREAD

**Issue by pdenert** [leancodepl/patrol#2416]
Describe how to use `--dart-define-from-file` and add info how to use patrol.env in custom location using this option.

**Comment by kubicki-4human** [comment]
Does simple `--dart-define` works for you?

**Comment by pdenert** [comment]
Yes `--dart-define` should work. If you have some trouble with using Patrol, join our discord (https://discord.gg/ukBK5t4EZg) and create new thread in #help-needed

**Comment by DmytroChuchmaiBarges** [comment]
@pdenert 

in my case I use 
```
abstract class EnvProps {
  static const _myEnvVar = String.fromEnvironment("myEnvVar");
....
}
```
--dart-define-from-file=".env.json"
and basically my file: 

{
"myEnvVar":"someSecretValue"
}

this is useful for passing apiUrl, some secret tokens that are being used for the build of the app itself
